stdlib/universe/quantile.go

```diff
@@ -212,14 +220,17 @@ func createQuantileTransformation(id execute.DatasetID, mode execute.Accumulatio
 	agg := &QuantileAgg{
 		Quantile:    ps.Quantile,
 		Compression: ps.Compression,
+		digest:      tdigest.NewWithCompression(ps.Compression),
 	}
+	_ = a.Allocator().Account(tdigest.ByteSizeForCompression(agg.Compression))
 	t, d := execute.NewAggregateTransformationAndDataset(id, mode, agg, ps.AggregateConfig, a.Allocator())
 	return t, d, nil
 }
+
 func (a *QuantileAgg) Copy() *QuantileAgg {
 	na := new(QuantileAgg)
 	*na = *a
-	na.digest = tdigest.NewWithCompression(na.Compression)
+	na.digest.Reset()
```

Comment:
I think it's probably fine, but this `Copy` method will change state of the original thing being copied, in order to avoid doing a deep copy of `a`. It seems like `Copy` is really a misnomer. That is not something that I would expect you to fix here; but it would be good to call out in a comment that this method does not really have copy semantics as one might reasonably expect.

Reply:
Also, it would be good to really be sure that instances produced by `QuantileAgg.Copy` are only used serially, since they will all share the same `tdigest` data structure. I believe that this is the case, since the aggregate `Process` method has only one input and will only process one table at a time.

Reply:
Did a global search in flux, and the only place `NewFloatAgg` (which wraps `Copy`) is [here](https://github.com/influxdata/flux/blob/2c5db92719a994e45de0c9026e6fe638a1d2f1b3/execute/aggregate.go#L108). I think as long as it stays that way, this implementation of Copy should be fine.

But maybe it would be worthwhile to rename it?

Reply:
If renaming it isn't too troublesome, then I think it's a good idea. If not, then it's okay to skip it.

Reply:
Done.